docs/docsite/rst/user_guide/intro_dynamic_inventory.rst

```diff
@@ -113,6 +113,8 @@ Inventory script example: AWS EC2
 
 If you use Amazon Web Services EC2, maintaining an inventory file might not be the best approach, because hosts may come and go over time, be managed by external applications, or you might even be using AWS autoscaling. For this reason, you can use the `EC2 external inventory  <https://raw.githubusercontent.com/ansible-collections/community.aws/master/scripts/inventory/ec2.py>`_ script.
 
+.. note:: If you install ansible by pip because probably you are in MacOS you may need to change the `#!/usr/bin/env python` in the `ec2.py` to `#!/usr/bin/env python3` so it will use Python3 instead of Python2.7.
+
```

Comment:
This seems a little overly specific, and makes a number of assumptions. 

In addition to this, we do not recommend editing any of the files included with Ansible.

This might indicate the need for an FAQ entry that describes how to generally manage python packages, and determining what python versions are in use.

Reply:
Can you help me to fix those specific things?

That .py script is not included in ansible, the guide ask you to download it

Sure, a FAQ entry could come handy

Reply:
@albertorm95 `ec2.py` inventory script is now deprecated in favor of [`aws_ec2`](https://github.com/ansible-collections/amazon.aws/blob/main/plugins/inventory/aws_ec2.py). Also, `ec2.py` references are being removed via https://github.com/ansible/ansible/pull/69107.